server/src/main/java/org/elasticsearch/reservedstate/service/ReservedStateUpdateTask.java

```diff
@@ -100,6 +101,36 @@ protected ClusterState execute(final ClusterState currentState) {
             }
         }
 
+        checkAndThrowOnError(errors, currentState, reservedStateVersion);
+
+        // Run the postTransform loop for any handlers that need to perform updates to other things than the cluster state
+        for (var handlerName : orderedHandlers) {
+            ReservedClusterStateHandler<?> handler = handlers.get(handlerName);
+            try {
+                // We first try to fetch the existing keys from what we have in the builder if transform saved any keys
```

Comment:
Nit: the word existing seems a bit confusing since there is a variable called `existingMetadata` and it is not the same thing. Would it be better to say `.. fetch the current keys ...".